### DISCUSSION TITLE
Two-dimentional simulation

### DISCUSSION THREAD

**Discussion by DOGzym** [firemodels/fds#13818] [Q&A]
Hi all,
In the case of setting up a two-dimentional model (I set the number of grids in the y direction to 1), will the grid size in the y direction affect the output results?

**Comment by drjfloyd** [comment]
Only volume integrated outputs will change; i.e., if you go from 1 mm to 1 cm in the y-direction your HRR will increase by a factor of 10 since  each grid cell will increase in volune by a factor of 10.  The HRRPUV would be the same.
